docs/feature-gates.md

```diff
@@ -341,3 +342,15 @@ More documentation will be coming in the future.
 
 User should provide a file server with this feature, and store its authentication credential in a Secret. Antrea
 Controller is required to be configured with the permission to read the Secret.
+
+### Multicluster
+
+The `Multicluster` feature gate of Antrea Agent enables [Antrea Multi-cluster Gateways](multicluster/user-guide.md#multi-cluster-gateway-configuration)
+which route Multi-cluster Service and Pod traffic through tunnels across clusters. The `Multicluster` feature gate
+of Antrea Controller enables support for [cross-cluster NetworkPolicy](multicluster/user-guide.md#networkpolicy-for-cross-cluster-traffic).
```

Comment:
Reading it again. Maybe change "cross-cluster" to "multi-cluster".

Reply:
Done